app/models/concerns/parameterizable.rb

```diff
@@ -0,0 +1,24 @@
+# handle converting ActiveModel attributes into a command line string
+# used in PAPI jobs via PapiClient and IngestJob
+module Parameterizable
+  extend ActiveSupport::Concern
+
+  # regular expression to validate GS url format
+  GS_URL_REGEXP = %r{\Ags://}.freeze
+
+  # convert attribute name into CLI-formatted option
+  def self.to_cli_opt(param_name)
+    "--#{param_name.to_s.gsub(/_/, '-')}"
+  end
+
+  # return array of all initialized attributes in Python CLI form, e.g. annotation_name => --annotation-name
```

Comment:
Hyphen-delimiters are a general CLI convention.

```suggestion
  # return array of all initialized attributes as CLI arguments, e.g. annotation_name => --annotation-name
```

Reply:
[81763b0](https://github.com/broadinstitute/single_cell_portal_core/pull/1634/commits/81763b0d5ce57fbb1355cce3b9e39c97052af5c4)